examples/travis.example.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - 4
+  - 5
+
+addons:
+  firefox: 'latest'
```

Comment:
I've been thinking, since we are in the process. Can we make it that the tests run both in Firefox and Chrome inside travis? It will be eventually needed when we get into WebRTC all the things.